ci/build/build-docker-images.py

```diff
@@ -770,9 +769,10 @@ def push_readmes(merge_build: bool):
 MERGE = "MERGE"
 HUMAN = "HUMAN"
 PR = "PR"
+RELEASE_PR = "RELEASE-PR"
```

Comment:
maybe just use also use underscore? so that it is the same as the var name?

Reply:
fixed!